service/raft/raft_group_registry.cc

```diff
@@ -151,6 +151,11 @@ void raft_group_registry::init_rpc_verbs() {
     auto handle_one_way_rpc = [this] (
             const rpc::client_info& cinfo,
             const raft::group_id& gid, raft::server_id from, raft::server_id dst, auto handler) {
+        if (*_my_id != dst) {
```

Comment:
Please replace `*_my_id` with `get_my_raft_id()`, which asserts that `_my_id` is initialized.
Same below.

Reply:
Replaced.